src/js/components/FileInput/doc.js

```diff
@@ -0,0 +1,142 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { getAvailableAtBadge } from '../../utils/mixins';
+import { themeDocUtils } from '../../utils/themeDocUtils';
+
+export const doc = FileInput => {
+  const DocumentedFileInput = describe(FileInput)
+    .availableAt(getAvailableAtBadge('FileInput'))
+    .description('A control to input one or more files.')
+    .usage(
+      `import { FileInput } from 'grommet';
+<FileInput id='file' name='file' />`,
+    )
+    .intrinsicElement('input');
+
+  DocumentedFileInput.propTypes = {
+    accept: PropTypes.string.description(
+      'MIME type pattern to match against. For example: "image/*".',
+    ),
+    aggregateThreshold: PropTypes.number
+      .description(
+        `The maximum number of individual files to show. Above this,
+      only a single message describing the number of files will be shown.`,
+      )
+      .defaultValue(10),
+    disabled: PropTypes.bool
+      .description('Whether the control is disabled.')
+      .defaultValue(false),
```

Comment:
I think the default value is `undefined`

Reply:
yep, fixed